packages/platform/platform-fastify/test/test-session.integration.spec.ts

```
@@ -0,0 +1,117 @@
+  email: string;
+
+  @Required()
+  @MinLength(6)
+  @Allow(null)
+  password: string;
+}
+
+export class User extends UserCreation {
+  @Ignore()
+  declare password: string;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)